highlight/HighlightsFromPointsExplainer.md

```diff
@@ -11,61 +11,529 @@ This document is intended as a starting point for engaging the community and sta
+This whole sentence is what Bob has selected on his.</br>
+</br>
+Clicking on those highlighted pieces of text shows which user selected them and intensifies the background color.</br> 
+</div>
+<div id="cursorBox"></div>
+<script>
+    let cursorBox = document.getElementById('cursorBox');
+    let div = document.querySelector('div');
+    let aliceTextNode = div.childNodes[4];
+    let aliceRange = new StaticRange({startContainer: aliceTextNode, startOffset: 0, endContainer: aliceTextNode, endOffset: 18});
+    let bobTextNode = div.childNodes[6];
+    let bobRange = new StaticRange({startContainer: bobTextNode, startOffset: 0, endContainer: bobTextNode, endOffset: bobTextNode.textContent.length});
+
+    let aliceHighlightName = 'selection-highlight-Alice';
+    let aliceHighlight = new Highlight(aliceRange);
+    let aliceActiveHighlight = new Highlight(aliceRange);
+    CSS.highlights.set(aliceHighlightName, aliceHighlight);
+    let bobHighlightName = 'selection-highlight-Bob';
+    let bobHighlight = new Highlight(bobRange);
+    let bobActiveHighlight = new Highlight(bobRange);
+    CSS.highlights.set(bobHighlightName, bobHighlight);
+
+    let highlightToUsername = new Map([
+        [aliceHighlight, "Alice"],
+        [bobHighlight, "Bob"]
+    ]);
+
+    function resetActiveHighlights() {
+        for (username of highlightToUsername.values()) {
+            let activeHighlightName = 'active-selection-highlight-' + username;
+            CSS.highlights.delete(activeHighlightName);
+        }
+        cursorBox.style.display = 'none';
+        cursorBox.textContent = '';
+    }
+
+    function setActiveHighlight(highlight, username, x, y) {
+        let activeHighlightName = 'active-selection-highlight-' + username;
+        CSS.highlights.set(activeHighlightName, highlight);
+        cursorBox.textContent += (cursorBox.textContent.length ? ', ' : '') + username;
+        cursorBox.style.display = 'block';
+        cursorBox.style.left = (x + 2) + 'px';
+        cursorBox.style.top = (y - cursorBox.getBoundingClientRect().height - 6) + 'px';
+    }
+
+    function createActiveHighlights(x, y) {
+        resetActiveHighlights();
+        let highlights = CSS.highlights.highlightsFromPoint(x, y);
+        for (highlight of highlights) {
+            let username = highlightToUsername.get(highlight);
+            if (username != undefined) {
+                setActiveHighlight(highlight, username, x, y);
+            }
+        }
+    }
+
+    document.addEventListener('mousemove', (event) => {
+        cursorBox.style.display = 'none';
+    });
+
+    document.addEventListener('click', (event) => {
+        createActiveHighlights(event.pageX, event.pageY);
+    });
+</script>
+</body>
+</html>
+```
+
+### Performance Analysis
+
+We compared the performance of the two approaches presented for the online collaboration example analyzed throughout this explainer: with and without the proposed `highlightsFromPoint` API.

```

Comment:
Not blocking for this PR, but it would be nice if you could post the full test page somewhere that people could reproduce this analysis for themselves.

Reply:
You mean post it in https://jsfiddle.net/ and include the link in the explainer for example? Or just an .html file in this repo?

Reply:
I have a slight preference for putting it in this repo next to the explainer. It's nice to have it together in one place, and for  a performance test better to have it in a standalone file rather than running in an `<iframe>` on jsfiddle where it's more likely that other stuff on the page could affect the performance.